arm-datalake-analytics/job/2016-03-20-preview/swagger/job.json

```diff
@@ -0,0 +1,1017 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DataLakeAnalyticsJobManagementClient",
+    "description": "Creates an Azure Data Lake Analytics job client.",
+    "version": "2016-03-20-preview",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true,
+      "internalConstructors": true
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{accountName}.{adlaJobDnsSuffix}",
```

Comment:
accountName will be added in the beginning by default. Please specify the positionInOperation: last.

Take a look at the documentation over here: https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-parameterized-host

I think, when the extension was designed initially, it was adding the path parameter in the end. Hence, when the extension was updated, datalake specs were updated accordingly. This seems like a new spec.